person/person.go

```diff
@@ -1,10 +1,15 @@
 package person
 
 type Person struct {
+	Id        int
 	FirstName string
 	LastName  string
 }
 
+func (p Person) GetId() int {
```

Comment:
According to [this](https://golang.org/doc/effective_go#Getters), getters in Go should leave out the "get" part, e.g. `Id()`

Reply:
Sorted